### ISSUE TITLE
A new version of LHR is possibly out. (MSI RTX 3080 Ti Gaming X Trio, lower hashrate than expected)

### ISSUE THREAD

**Issue by szczesniak-piotr** [develsoftware/GMinerRelease#439]
Hi everyone, especially @develsoftware. 

I bought an RTX 3080 Ti a few days ago. What I noticed quickly was that I could not get it to the hash rate it was supposed to do. The difference is not very big, but it's there. I got to 77-79 mhs and cannot go higher. Other 3080 Ti doing 80+ ez. I contacted some people and it turned out that some of them have the same issue with the card. I tried everything, from drivers, through bios, to OC and different miners, there is no possible way to get this card higher. The power draw is stuck around 220-240W and it just will not go up. I'm bringing this up so everyone knows it is possible we got another version of LHR. I've also read some threads on Reddit from ppl claiming their newly bought LHR cards performed worse than the other ones (also LHR). Have a good one guys.

**Comment by szczesniak-piotr** [comment]
@evark1 
I've got MSI Gaming X Trio. Memory temperature is ~86-88c. I'm currently on Windows. Card settings are not important, I've tried everything from -500 core to +100 core, locking the core from 1150 to 1500, memory anywhere from +600 to +1600, power from 75% to 110%, combinations of these including, different miners as well. Now I'm getting ~79mhs around 240W, power 100, core 0, memory +1300.
